Bottom: e102d5fd0fd35632eeafc6f3d2b1ffea21873167
Top:    c9a2a77be772b959e4810744bb170212d1c48274
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-27 14:49:35 -0700

Refresh of rs-states

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 9beeddc..8cbd425 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -122,7 +122,7 @@ struct rs_conn_data {
 };
 
 /*
- * rsocket states are ordered as connecting, connected, disconnected.
+ * rsocket states are ordered as passive, connecting, connected, disconnected.
  */
 enum rs_state {
 	rs_init,
